tools/render-test/render-d3d11.cpp

```diff
@@ -201,102 +195,88 @@ Renderer* createD3D11Renderer()
 
     // The `D3DCompile` entry point takes a bunch of parameters, but we
     // don't really need most of them for Slang-generated code.
-    ID3DBlob* dxShaderBlob = nullptr;
-    ID3DBlob* dxErrorBlob = nullptr;
-    HRESULT hr = D3DCompile_(
-        source,
-        strlen(source),
-        sourcePath,
-        &defines[0],
-        nullptr,
-        entryPointName,
-        dxProfileName,
-        flags,
-        0,
-        &dxShaderBlob,
-        &dxErrorBlob);
+    ID3DBlob* shaderBlob = nullptr;
+    ID3DBlob* errorBlob = nullptr;
+    HRESULT hr = compileFunc(source, strlen(source), sourcePath, &defines[0], nullptr, entryPointName, dxProfileName, flags, 0,
```

Comment:
More rationale: my personal convention is that as soon as an argument list gets complex to follow as a one-liner, I always break it into the one-argument-per-line form, because:

- It supports adding per-argument comments in case clarification is needed
- Any argument list that long will almost certainly involve changes down the line, and putting each argument on its own line helps make any diffs at call sites easier to read (a lot of my personal convention is around making diffs easy to read...)

Anyway, I'm sorry to keep pointing out these things but I just want to note that some of the stuff being cleaned up here had an underlying rationale of its own. I'm 100% okay with cleaning up messy code as you go through and add new stuff to part of the system, but I also want to make sure that all of us in the project don't end up in some kind of "edit war" where we constantly rewrite each other's code to match our personal aesthetics.